doc/6-Performance.md

```diff
@@ -321,4 +321,39 @@ After `cleaner`:
 |                 2500ms | 5218| 5318|  446|
 
 
+
```

Comment:
Thanks a lot @skai111 for collecting and sharing all that data. It's extremely valuable.

Could you just move your table above the `<!--- !DMNONE --->` comment? The `DMNONE` sections in the help are (intentionally) not copied to the calculator help file, because the built-in help viewer cannot display tables.

Reply:
Hopefully I got it right  and changed it in the right place.